### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by mandrews**

### CODE REVIEW

File: lib/masamune/thor.rb [hunk 1]
```diff
@@ -149,7 +149,7 @@ def initialize(_args = [], _options = {}, _config = {})
             end
           end
 
-          after_initialize_invoke(options.symbolize_keys)
+          after_initialize_invoke(options.to_hash.symbolize_keys)
```
Comment (mandrews) [inline lib/masamune/thor.rb]: `Thor::HashWithIndifferentAccess` is behaving strangely after call to `symbolize_keys`- convert to plain old `Hash`

File: spec/masamune/filesystem_spec.rb [hunk 2]
```diff
@@ -994,7 +994,7 @@
     context 'hdfs dir' do
       it do
         expect(filesystem).to receive(:root_path?).once.and_return(false)
-        expect(filesystem).to receive(:hadoop_fs).with('-rmr', 'file://' + old_dir)
+        expect(filesystem).to receive(:hadoop_fs).with('-rm', '-r', 'file://' + old_dir)
```
Comment (mandrews) [inline spec/masamune/filesystem_spec.rb]: `hadoop fs -rmr` is deprecated in favor of `hadoop fs -rm -r`

File: lib/masamune.rb [hunk 3]
```diff
@@ -43,13 +43,4 @@ module Masamune
...
   extend Masamune::HasEnvironment
-
-  def load(config_file, &block)
-    Masamune::Environment.new.tap do |env|
-      env.configure do |config|
-        config.load(config_file)
-      end
-      env.catalog.instance_eval &block
-    end
-  end
```
Comment (mandrews) [inline lib/masamune.rb]: remove dead code

**Comment by phene** [comment]
+1

**Pull request merged**